src/mesh/vtk_helper.jl

```diff
@@ -124,6 +124,90 @@ function quad_vtk_order(corner_verts, order, dim, skip = false)
     return coords
 end
 
+"""
+    hex_vtk_order(corner_verts, order, dim, skip = false)
+
+Compute the coordinates of a VTK_LAGRANGE_HEXAHEDRON of a hex of order `order`
+defined by the coordinates of the vertices given in `corner_verts`. `dim` is
+the dimension of the coordinates given. If `skip` is set to true, the
+coordinates of the vertex- and edge-points aren't computed, which can be used
+to compute points of a `VTK_LAGRANGE_HEXHEDRON`
+Inspired by: https://github.com/ju-kreber/paraview-scripts/blob/master/node_ordering.py
+"""
+function hex_vtk_order(corner_verts, order, dim, skip = false)
+
+    @assert order >= 0 "`order` must be non-negative."
+    
+    coords = copy(corner_verts)
+    if order == 0
+        return coords
+    end
+    
+    # Edges.
+    num_verts_on_edge = order - 1
```

Comment:
`verts` here refers to nodes on a given edge, right? I noticed the other routines also use this terminology. Would you mind doing a search-and-replace for the other instances too?
```suggestion
    num_nodes_on_edge = order - 1
```

Reply:
Done.